ts/DataGrid/Accessibility/Accessibility.ts

```diff
@@ -113,15 +120,15 @@ class Accessibility {
      * Load the accessibility options.
      */
     public loadOptions(): void {
-        const options = this.dataGrid.options?.accessibility;
-        if (!options) {
+        const lang = this.dataGrid.options?.lang?.accessibility;
+        if (!lang) {
```

Comment:
Actually is not lang, but langAccessibility ref.

There are 2 solutions:
1. use `const lang = this.dataGrid.options?.lang` and then `lang.accessibility` in the code
2. use `const langAccessibility = this.dataGrid.options?.lang?.accessibility`

Otherwise it's confusing.

Reply:
Its removed anyway in the https://github.com/highcharts/highcharts/pull/22082, so will not fixing it to avoid merge conflicts.